File: src/main/java/com/lord/small_box/dtos/PurchaseOrderItemDto.java
package com.lord.small_box.dtos;

import java.math.BigDecimal;
import java.util.Calendar;

import com.lord.small_box.models.PurchaseOrder;

import lombok.Getter;
import lombok.Setter;


@Setter
@Getter
public class PurchaseOrderItemDto {
private Long id;
	
	private String code;
	
	private String programaticCat;
	
	private int quantity;
	
	private String measureUnit;
	
	private String itemDetail;
	
	private BigDecimal unitCost;
	
	private BigDecimal  estimatedCost;
	
	private BigDecimal totalEstimatedCost;
	
	private Calendar expirationDate;
	
	
}

File: front/small-box/src/app/models/purchaseOrderItemDto.ts
export class PurchaseOrderItemDto{
    code!:string;
    programaticCat!:string;
    quantity!:number;
    measureUnit!:string;
    itemDetail!:string;
    unitCost!:number;
    estimatedCost!:number;
    totalEstimatedCost!:number;
    expirationDate!:Date;
}